src/neptune/handler.py

```diff
@@ -362,15 +361,15 @@ def append(
             * `FileSeries` - series of files
 
         When you log the first value, the type of the value determines what type of field is created.
-        For more, see the field types documentation: https://docs.neptune.ai/api/field_types
+        For more information, see the field types document: https://docs.neptune.ai/api/field_types
```

Comment:
I think we don't generally use the word _document_. For this, we can use "documentation" or "reference". Or, perhaps even better, something like "To learn more about field types, see the docs".